cfpb/cfgov-refresh
# Ask CFPB: QA fixes round 2

Ask style updates based on QA


## Changes

- Increase legal disclaimer bottom padding on Audience pages
- Hide feedback module for print on Spanish Answer pages
- Add padding to English Landing page banner at medium screen sizes
- `Legal disclaimer` reusable text title is hidden
- Top margin added to English landing page category section on mobile to fix spacing with hero image

## Testing

- Check out branch and run `gulp styles`
- Verify that 
  - `About us` section on [landing page](http://localhost:8000/ask-cfpb/) has left/right padding on medium screen sizes (between 900-1000px)
   - Legal disclaimer has bottom padding on [audience pages](http://localhost:8000/ask-cfpb/audience-servicemembers/)
  - Print preview on [Spanish answer page](http://localhost:8000/es/obtener-respuestas/que-efecto-tendria-sobre-mi-credito-comparar-diferentes-ofertas-de-prestamos-para-vehiculos-es-763/) doesn't show feedback module
  - `Legal disclaimer` reusable text title is hidden

## Screenshots
### Audience page Legal disclaimer padding:

<img width="964" alt="screen shot 2017-06-11 at 11 05 59 pm" src="https://user-images.githubusercontent.com/778171/27021052-b77b2052-4efa-11e7-8efd-4a6b6ead4d49.png">


### Landing page banner padding at medium screen sizes:
<img width="896" alt="screen shot 2017-06-11 at 11 04 59 pm" src="https://user-images.githubusercontent.com/778171/27021070-d29de194-4efa-11e7-8b25-b46c553c9c22.png">


### Spanish Answer page feedback module hidden on print:
<img width="507" alt="screen shot 2017-06-11 at 11 06 16 pm" src="https://user-images.githubusercontent.com/778171/27021084-ef1c68ae-4efa-11e7-8a96-613207096a7a.png">

### English legal disclaimer title is hidden
<img width="840" alt="screen shot 2017-06-12 at 9 48 44 am" src="https://user-images.githubusercontent.com/778171/27045139-ebcae284-4f54-11e7-836c-0166874cba79.png">

### Hero spacing on mobile landing page
<img width="440" alt="screen shot 2017-06-12 at 9 38 28 am" src="https://user-images.githubusercontent.com/778171/27045130-e61765ec-4f54-11e7-84c1-c49d290b8ced.png">


## Checklist
